apps/react/src/features/anime/components/AnimeSidebar/AnimeSidebar.tsx

```diff
@@ -0,0 +1,28 @@
+import { ListItem } from '@mui/material';
+
+import { FC, memo } from 'react';
+
+import { TabGroup } from '../../../../components';
+
+import { AnimeInfiniteScroll } from '../AnimeInfiniteScroll/AnimeInfiniteScroll';
+import { AnimeSearch } from '../AnimeSearch/AnimeSearch';
+
+import style from './AnimeSidebar.module.css';
+
+export const AnimeSidebarInner: FC = () => (
+  <ListItem className={style['anime-sidebar']}>
```

Comment:
Since it's a sidebar, let's be more semantically correct
```suggestion
  <ListItem component="aside" className={style['anime-sidebar']}>
```

Reply:
BTW, why it's `ListItem`? Maybe it should be `List` instead?